docs/_guide/styles.md

```diff
@@ -111,27 +111,33 @@ To define a static `styles` property:
 
 #### Expressions in static styles
 
+The `css` tag only accepts literal strings. You can include expressions in your static styles using the `unsafeCSS` tag instead. 
+
 Static styles apply to all instances of an element. Any expressions in your CSS are evaluated and included **once**, then reused for all instances. 
 
-For security reasons, use **unsafeCSS** to pass non-literal values using only trusted sources:
+Example:
 
 ```js
-import {LitElement, css, unsafeCSS} from 'lit-element';
-
-const displayType = css`block`;
-const mainColor = unsafeCSS('#a50e0e');
+import { LitElement, unsafeCSS } from 'lit-element';
 
 class MyElement extends LitElement {
   static get styles() {
-    return css`
-    :host {
-      display: ${displayType};
-      color: ${mainColor};
-    }`;
+    return unsafeCSS`
```

Comment:
I don't think you can use unsafeCSS this way. You can only use it as a filter inside an expression.

For example:

```
css`
  :host {
    color: ${unsafeCSS(mainColor)};
  }`;
```

Reply:
i.e., it's not a tag function like `css`, it's a method you can use in expressions.

Reply:
yep sorry I merged too soon. fixed it